MenMa WatchKit Extension/InterfaceController.swift

```diff
@@ -58,11 +58,13 @@ class InterfaceController: WKInterfaceController, WCSessionDelegate {
         
         let messageToSend = ["value": "message sent successfully"]
         session.sendMessage(messageToSend, replyHandler: { replyMessage in
-            //            //handle and present the message on screen
-            //            let value = replyMessage["value"] as? Venue
-            //            //            self.messageLabel.setText(value)
+            //handle and present the message on screen
+            let value = replyMessage["value"] as? Venue
```

Comment:
@lynneokada About sending venues data back and forth between iOS and watch, here is how it works:

1. From the watch, session.sendMessage:replyHandler:errorHandler: is called to ask the iOS to download venues data from the web.
2. In the iOS, the delegate method session:didReceiveMessage:replyHandler: is called. In this method, the iOS downloads venues data if it's empty, and encode it. Then, call session.sendMessage:replyHandler:errorHandler: method to send venues to the watch.
3. In the watch, the delegate method session:didReceiveMessageData:replyHandler: is called. Decode venues data, and pass it to configureRamenTableView: method of the watch's tableViewController.

Note that both the iOS and the watch use session.sendMessage(Data), and use the delegate method session:didReceiveMessage(Data):replyHandler: to communicate with each other. And, basically we don't need to use replyHandler.

Hope my explanation is understandable...